### PR TITLE
[php/libraries/NDB_Menu_Filter.class.inc] filter undefined fix

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Fixes this error I received today when visiting the user_accounts module.
`Notice: Undefined index: filter in /Users/alizee/Development/GitHub/McGill/Loris/php/libraries/NDB_Menu_Filter.class.inc on line 598`

#### Testing instructions (if applicable)

1. Visit the user_accounts module on fresh rasinbread installation with only admin account.
2. Check error_log

### CODE REVIEW

File: php/libraries/NDB_Menu_Filter.class.inc [hunk 1]
```diff
@@ -595,7 +595,7 @@ class NDB_Menu_Filter extends NDB_Menu
         $this->setTemplateVar('rowsPerPage', $rowsPerPage);
         $this->setTemplateVar('pageID', $pageID);
 
-        if (is_array($_GET['filter']['order'])) {
+        if (isset($_GET['filter']) && is_array($_GET['filter']['order'])) {
```
Comment (driusan) [inline php/libraries/NDB_Menu_Filter.class.inc]: `$_GET['filter']` needs to be checked to be an array, not just set.

Comment (maltheism) [reply php/libraries/NDB_Menu_Filter.class.inc]: @driusan yah both are checked. But currently I can get an error since it can be undefined.

Comment (driusan) [reply php/libraries/NDB_Menu_Filter.class.inc]: Both aren't checked. `$_GET['filter']` is only checked to be set, `$_GET['filter']['order']` is checked to be an array. `$_GET['filter']` needs to be both set and an array for `$_GET['filter']['order']` to be valid regardless of whether or not `$_GET['filter']['order']` is an array.

**Comment by driusan** [comment]
Closing since @maltheism approved #6047

**Pull request closed without merging**